function.tf

```diff
@@ -38,10 +38,23 @@ resource "google_pubsub_topic_iam_member" "cloudfunction_pubsub" {
   member = "serviceAccount:${google_service_account.cloudfunction[0].email}"
 }
 
+resource "google_storage_bucket" "this" {
+  name     = "${var.name}-bucket"
+  location = "US"
+
+  force_destroy = true
```

Comment:
This is typically not a good default when you're making durable datastores for users. What potentially makes this an exception is the fact that the bucket is essentially just a temporary buffer for the exported data.

_This_ would be a good thing to clarify in a code comment, e.g.

> Since the bucket is just a temporary storage for asset export objects until the function can process them, we want to implicitly delete any leftover objects if Terraform plans to remove the bucket